troupon/troupon/settings/development.py.orig

```diff
@@ -0,0 +1,135 @@
+"""
+Django settings for troupon project.
+
+Generated by 'django-admin startproject' using Django 1.8.3.
```

Comment:
Will go through them and remove them asap.
On Oct 13, 2015 17:15, "Chidiebere Nnadi" <notifications@github.com> wrote:

> In troupon/troupon/settings/development.py.orig
> <https://github.com/andela/troupon/pull/46#discussion_r41888620>:
>
> > @@ -0,0 +1,135 @@
> > +"""
> > +Django settings for troupon project.
> > +
> > +Generated by 'django-admin startproject' using Django 1.8.3.
>
> I don't know why this file is here too...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/andela/troupon/pull/46/files#r41888620>.
>